OpenDreamRuntime/Procs/DMOpcodeHandlers.cs

```diff
@@ -420,7 +420,8 @@ private static void HandleSuffixPronoun(ref StringBuilder formattedString, ReadO
                     if (listObject.IsSubtypeOf(state.Proc.ObjectTree.Atom) || listObject.IsSubtypeOf(state.Proc.ObjectTree.World)) {
                         list = listObject.GetVariable("contents").GetValueAsDreamList();
                     } else {
-                        throw new Exception($"Value {listObject} is not a {state.Proc.ObjectTree.List}, {state.Proc.ObjectTree.Atom}, or {state.Proc.ObjectTree.World}");
+                        // BYOND ignores all floats, strings, types, etc. here and just doesn't run the loop.
```

Comment:
This code isn't related to loops.
```suggestion
                        // BYOND ignores all floats, strings, types, etc. here and just returns 0.
```